Sources/ArcGISToolkit/Documentation.docc/Tutorials/JobManagerTutorial.tutorial

```diff
@@ -0,0 +1,138 @@
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep3, previousFile: JobManagerViewStep2) {
+                    @Image(source: JobManagerTutorial, alt: "An Image of an iPhone app with a progress bar and Start New Job button.")
+                }
+            }
+            
+            @Step {
+                Let's look at what we have so far. When the "Take Map Offline" button is tapped, a job is added to the job manager and the job is started.
+                
+                The first thing the job does is connect to the specified server to kick off the process that will make a mobile map package. Jobs are intended to be used with longer running service tasks. Once the task on the server is started the job waits and polls.
+
+                @Image(source: JobManagerTutorialJobStart, alt: "An image of the app UI with a progress bar four percent done.")
+            }
+            
+            @Step {
+                To know when the work on the server is done, the job polls the service. The job periodically requests the status of the work the service is doing.
+                
+                For this tutorial, the service is collecting all the parts of a map and putting them together to create a mobile map package.
+                
+                @Image(source: JobManagerTutorialJobPoll, alt: "An image of the app UI with a progress bar twenty-one percent done.")
+            }
+            
+            @Step {
+                Eventually, the job will poll the service and receive the response that the work is complete and there is a result available to download. At this point the job starts a URL session and downloads the result.
+                
+                The mobile map package is downloaded from the server to a local folder.
+                
+                @Image(source: JobManagerTutorialJobDownload, alt: "An image of the app UI with a progress bar one hundred percent done.")
+            }
+            
+            @Step {
+                To generate the offline map you will need an API key or credential. This step show how to add an API key. To use a credential refer to the AuthenticatorTutorial.
+                
+                Once an API key has been added the project can be built and run. Tap the "Take Map Offline" button to add the job to the job manager and start the job. The offline map will be generated and downloaded. As long as the tutorial app is running in the foreground the job will finish and we'll get the mobile map package. However, we know that people don't use their phones like that. What we want is for people to be able to start a job, do other things on their phone while the job runs and get a notification when the job finishes. In the next steps you will add the code to handle the app being backgrounded or terminated.
```

Comment:
Should we generalize this to "device" rather than just "phone"s? I get a sense that iPads are fairly common platforms for apps using our SDK.

```suggestion
                Once an API key has been added the project can be built and run. Tap the "Take Map Offline" button to add the job to the job manager and start the job. The offline map will be generated and downloaded. As long as the tutorial app is running in the foreground the job will finish and we'll get the mobile map package. However, we know that people don't use their devices like that. What we want is for people to be able to start a job, do other things on their device while the job runs and get a notification when the job finishes. In the next steps you will add the code to handle the app being backgrounded or terminated.
```